baggage/templates/baggage_list.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base_table.html" %}
+{% load django_tables2 %}
+{% block head_title %}All baggage{% endblock %}
+
+{% block table_footer %}
+{% endblock %}
```

Comment:
No need to load django_tables2 again. See: https://github.com/hackupc/registration/blob/master/checkin/templates/checkin/list.html